autocue2.liq

```diff
@@ -5,416 +5,389 @@
 
 # --- Copy-paste Azuracast LS Config, second input box BEGIN ---
 
-# Initialize settings for autocue2 protocol
-let settings.autocue2 = ()
+# Initialize settings for cue_file autocue implementation
+let settings.autocue.cue_file = ()
 
-let settings.autocue2.path =
+let settings.autocue.cue_file.path =
   settings.make(
     description=
-      "Path of the autocue2 binary.",
+      "Path of the cue_file binary.",
     "cue_file"
   )
 
-let settings.autocue2.timeout =
+let settings.autocue.cue_file.fade_in =
   settings.make(
     description=
-      "Timeout (in seconds) for autocue2 executions.",
+      "Default fade-in duration if no specified by the user.",
+    0.1
+  )
+
+let settings.autocue.cue_file.fade_out =
+  settings.make(
+    description=
+      "Default fade-out duration if no specified by the user.",
+    3.
+  )
+
+let settings.autocue.cue_file.metadata_override =
+  settings.make(
+    description=
+      "Disable processing when one of these metadata is found",
+    [
+      "liq_cue_in",
+      "liq_cue_out",
+      "liq_cross_start_next",
+      "liq_fade_in_delay",
+      "liq_fade_out_delay"
+    ]
+  )
+
+let settings.autocue.cue_file.timeout =
+  settings.make(
+    description=
+      "Timeout (in seconds) for cue_file executions.",
     60.
   )
 
-let settings.autocue2.target =
+let settings.autocue.cue_file.target =
```

Comment:
A litlle more nesting does not hurt.

Reply:
Can the settings.make have an empty array? I _don’t_ want to disable processing, especially not on _one tag alone_. Maybe I’ll _skip_ if I see _all needed_ (from file tags).

Reply:
Not sure I understand this.